FordLabs/PeopleMover
# Add test to sort alphabetically

Co-authored-by: Alex Wojtala <alexwojtala@gmail.com>
Co-authored-by: chris dorman <cdorman1@ford.com>

## Issue
Resolves #326  

## What was done
- [x] Wrote cypress tests for sorting: alphabetically, by product tag and by location

## How to test
1. Run cypress tests
2. Observe new sort tests are running and passing
